lib/operations/command_v2.js

```diff
@@ -15,10 +15,18 @@ class CommandOperationV2 extends OperationBase {
   constructor(parent, options, operationOptions) {
     super(options);
 
+    const noInherit = options && options.noInherit;
```

Comment:
Now that you're using a wrapper method, can't we use "aspects" to skip the inheritance?